Cynosura.Studio.CliTool/Program.cs

```diff
@@ -20,9 +20,18 @@ public static async Task Main(string[] args)
                     Environment.ExitCode = 1;
                 }
             }
-            catch (Exception e)
+            catch (Exception exception)
             {
-                Console.WriteLine($"{e.GetType().Name}: {e.Message}");
+                var ex = exception;
+                var message = "";
+                while (ex !=null)
```

Comment:
Missing space here. Also why not using exception.ToString()?

Reply:
Added missing space. 
`exception.ToString()` may ignore InnerException

Reply:
Changed to `exception.ToString()`